controller/pokecontroller.go

```diff
@@ -30,10 +31,28 @@ func (pI *pokeController) GetPokemon(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusNotFound)
 		return
 	}
+	w.Header().Add("Content-Type", "Applicaiton/Json")
 	json.NewEncoder(w).Encode(p)
 }
 
 func (pI *pokeController) GetPokemons(w http.ResponseWriter, r *http.Request) {
-	p := pI.pokeInteractor.GetAll()
+	p, err := pI.pokeInteractor.GetAll()
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	w.Header().Add("Content-Type", "Applicaiton/Json")
```

Comment:
Found a typo! `Application/Json`. You would like to check your handlers to see if you missed some more

Reply:
Yeah I did!, fixed them on the third deliverable.